public/js/components/AtomList/AtomList.js

```diff
@@ -27,17 +37,46 @@ class AtomList extends React.Component {
     state = {
         params: {
             types:[],
-            "page-size": "20",
-            q: "",
-            searchFields: "data.title,data.label,title,labels,data.body"
+            'page-size': '20',
+            q: '',
+            searchFields: 'data.title,data.label,title,labels,data.body'
         }
     };
 
     componentWillMount() {
-        this.props.atomListActions.getAtomList(this.state.params);
+        this.getAtomList(this.props.location.query);
+    }
+
+    getAtomList = (query) => {
+        if (Object.keys(query).length === 0) {
+          return this.props.atomListActions.getAtomList(this.state.params);
+        }
+
+        const convertUriListToArray = (uriList) => {
+          if (uriList.length === 0) {
+            return false;
+          }
+
+          return uriList.split(',');
+        };
+
+        const newParams = Object.assign({}, this.state.params, {
```

Comment:
Do you need to de-url-encode this? will this result in the url encoded versions being re-url-encoded